# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../fusionsound/DFBResult-hotfix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Use DFBResult as defined in the function prototypes in fusionsound.h

--- FusionSound-1.1.1/src/fusionsound.c.orig	2010-08-03 14:22:04.584253866 +0200
+++ FusionSound-1.1.1/src/fusionsound.c	2010-08-03 14:25:57.932007684 +0200
@@ -116,7 +116,7 @@
      return fs_config_set( name, value );
 }
 
-DirectResult
+DFBResult
 FusionSoundCreate( IFusionSound **ret_interface )
 {
      DFBResult ret;
@@ -160,8 +160,8 @@
      return ret;
 }
 
-DirectResult
-FusionSoundError( const char *msg, DirectResult error )
+DFBResult
+FusionSoundError( const char *msg, DFBResult error )
 {
      if (msg)
           fprintf( stderr, "(#) FusionSound Error [%s]: %s\n", msg, DirectResultString( error ) );
@@ -172,7 +172,7 @@
 }
 
 DFBResult
-FusionSoundErrorFatal( const char *msg, DirectResult error )
+FusionSoundErrorFatal( const char *msg, DFBResult error )
 {
      FusionSoundError( msg, error );
      
@@ -180,7 +180,7 @@
 }
 
 const char *
-FusionSoundErrorString( DirectResult error )
+FusionSoundErrorString( DFBResult error )
 {
      return DirectResultString( error );
 }
